### PR TITLE
Rigctl argument tweaks

### PR DESCRIPTION
**Pull request by Mashintime**
Tweaks to RTS and DCD arguments

### CODE REVIEW

**Comment by Mashintime** [comment]
@DJ2LS needs testing as I don't use these particular arguments.

File: gui/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "FreeDATA",
   "description": "FreeDATA Client application for connecting to FreeDATA server",
   "private": true,
-  "version": "0.13.4-alpha",
+  "version": "0.13.5-alpha",
```
Comment (DJ2LS) [inline gui/package.json:5]: We already have 0.13.5, we could increase to 0.13.6

File: modem/server.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 CORS(app)
 CORS(app, resources={r"/*": {"origins": "*"}})
 sock = Sock(app)
-MODEM_VERSION = "0.13.4-alpha"
+MODEM_VERSION = "0.13.5-alpha"
```
Comment (DJ2LS) [inline modem/server.py:32]: 0.13.6

**Review by DJ2LS — APPROVED** [review]

**Comment by Mashintime** [comment]
I rebased on main initially.

**Review by DJ2LS — APPROVED** [review]

**Review by DJ2LS — APPROVED** [review]

**Pull request merged**